app/views/root/designprinciples.html.erb

```diff
@@ -44,7 +44,8 @@
      <li class="principle">
       <article>
         <hgroup>
-          <h1 id="first">Start with user needs</h1>
+          <h1 id="first">Start with user needs*</h1>
+          <h2>*user needs not government needs</h2>
```

Comment:
Fair enough. I think there are definite benefits to losing the asterisk and making it part of the heading (e.g. avoiding the case detailed in https://github.com/alphagov/design-principles/pull/272#issuecomment-252977891), so I think it's worth exploring if we can at least do that, even if we do want to retain the repetition.